src/workers/notifications/squadFeaturedUpdated.ts

```diff
@@ -0,0 +1,37 @@
+import { NotificationType } from '../../notifications/common';
+import { generateTypedNotificationWorker } from './worker';
+import { NotificationSourceContext } from '../../notifications';
+import { SourceMemberRoles } from '../../roles';
+import { SourceMember } from '../../entity';
+import { In } from 'typeorm';
+
+const toNotify = [SourceMemberRoles.Admin, SourceMemberRoles.Moderator];
+
+const worker = generateTypedNotificationWorker<'api.v1.squad-featured-updated'>(
+  {
+    subscription: 'api.squad-featured-updated-notification',
+    handler: async ({ squad }, con) => {
+      if (!squad.flags.featured) {
```

Comment:
check if `flags` is JSON string here or object, not sure if this happens in CDC or in workers as well.

Reply:
Another inaccuracy with the CDC type, similar to the issue on the other PR 😭 

I tried to fix it, but more strict mode issues are coming up after the [changes](https://github.com/dailydotdev/daily-api/pull/2183/commits/1756964b2ee49e169da08a9e45ac19d319c6de03). Decided not to pursue yet 😿

Reply:
yes, good for now

Reply:
Oh I see you do the check here already...